File: payments/search/python/payments_by_reason_and_installments.py
# coding: UTF-8

import os, sys
import mercadopago

import json

def index(req, **kwargs):
    mp = mercadopago.MP("CLIENT_ID", "CLIENT_SECRET")
    
    filters = {
        "payment_type": "credit_card",
        "operation_type": "regular_payment",
        "installments": 12,
        "reason": "Your-item-title"
    }

    searchResult = mp.search_payment(filters)

    return json.dumps(searchResult, indent=4)

File: payments/search/nodejs/payments_by_reason_and_installments.js
var MP = require ("mercadopago");

var mp = new MP ("CLIENT_ID", "CLIENT_SECRET");

var filters = {
	"payment_type": "credit_card",
	"operation_type": "regular_payment",
	"installments": 12,
	"reason": "Your-item-title"
};

mp.searchPayment (filters, function (err, data){
	if (err) {
		console.log (err);
	} else {
		console.log (JSON.stringify (data, null, 4));
	}
});